applications/GeoMechanicsApplication/custom_constitutive/tension_cutoff.hpp

```diff
@@ -0,0 +1,40 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//
+//  Main authors:    Mohame Nabi,
+//                   Wijtze Pieter Kikstra
+// //
+
+#pragma once
+
+#include "custom_constitutive/yield_surface.h"
+
+namespace Kratos
+{
+
+class KRATOS_API(GEO_MECHANICS_APPLICATION) TensionCutoff : public YieldSurface
+{
+public:
+    KRATOS_CLASS_POINTER_DEFINITION(TensionCutoff);
+
+    TensionCutoff() = default;
+
+    explicit TensionCutoff(double tensileStrength);
```

Comment:
The parameter name doesn't comply with the Kratos Style Guide:
```suggestion
    explicit TensionCutoff(double TensileStrength);
```